api/index.py

```
@@ -0,0 +1,50 @@
+from http.server import BaseHTTPRequestHandler
+import json
+
+class handler(BaseHTTPRequestHandler):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)